### ISSUE TITLE
Forbid comments in `f` and `t` string

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#3404]
This code should raise a violation:

```python
>>> x = f'{ 1 # comment
... }'
```

Because comments make f string implicilty multiline.
And comments must not be present in strings. This is not right.

**Comment by LordGvozd** [comment]
Can I work on this?

**Comment by vesalius512** [comment]
Hello, can I work on this?
